lib/rubocop/rspec/cop_helper.rb

```diff
@@ -50,12 +50,17 @@ def autocorrect_source(source, file = nil)
   end
 
   def autocorrect_source_with_loop(source, file = nil)
+    cnt = 0
     loop do
       cop.instance_variable_set(:@corrections, [])
       new_source = autocorrect_source(source, file)
       return new_source if new_source == source
 
       source = new_source
+      cnt += 1
+      if cnt > RuboCop::Runner::MAX_ITERATIONS
+        raise RuboCop::Runner::InfiniteCorrectionLoop.new(file, [])
```

Comment:
Raise `InfiniteCorrectionLoop` in test also. Because `autocorrect_source_with_loop` will have infinite loop without this raising.